internal/pkg/cli/run_local.go

```diff
@@ -237,6 +239,16 @@ func newRunLocalOpts(vars runLocalVars) (*runLocalOpts, error) {
 		return nil
 	}
 	o.buildContainerImages = func(mft manifest.DynamicWorkload) (map[string]string, error) {
+		var dfDir string
+		if dockerWkld, ok := mft.Manifest().(dockerWorkload); ok {
```

Comment:
Do we need to have specific error message if the workload type does not implement `dockerWorkload` interface.
This could happen in case of static site  because we do not have dockerfile for this wkld.

And also I am curious what error msg do we display when deploying a `static site` with `run local` cmd.